cypress/integration/admin/pageBuilder/page/importExportPages.spec.js

```diff
@@ -1,129 +1,134 @@
-//             cy.findByText(/Show details/i).click();
-//             cy.findByTestId("import-pages-dialog.show-detail-list")
-//                 .children()
-//                 .should("have.length", 1);
-//
-//             cy.findByText(/Continue/i).click();
-//         });
-//
-//         // Page should be there
-//         searchForPage(pageTitle);
-//
-//         cy.findByTestId("default-data-list").within(() => {
-//             cy.get(".mdc-list-item")
-//                 .first()
-//                 .within(() => {
-//                     cy.findByText("Welcome to Webiny").should("exist");
-//                     cy.findByText(/Static/i).should("exist");
-//                     cy.findByText(/Draft/i).should("exist");
-//                     cy.findByText(/(v1)/i).should("exist");
-//                     cy.findByTestId("pages-default-data-list.select-page").click({ force: true });
-//                 });
-//         });
-//
-//         // Delete the imported page
-//         cy.findByTestId("default-data-list").within(() => {
-//             cy.get(".mdc-list-item")
-//                 .first()
-//                 .within(() => {
-//                     cy.findByTestId("pages-default-data-list.select-page").click({ force: true });
-//                 });
-//         });
-//         cy.findByTestId("pb-page-details-header-delete-button").click();
-//         cy.findByTestId("pb-page-details-header-delete-dialog").within(() => {
-//             cy.findByText(/Confirm/i).click();
-//         });
-//         cy.findByTestId("default-data-list").within(() => {
-//             cy.get(".mdc-list-item")
-//                 .first()
-//                 .within(() => {
-//                     cy.findByText(/Draft/i).should("not.exist");
-//                 });
-//         });
-//     });
-// });
+// We need this to grant Cypress access to the clipboard.
+Cypress.automation("remote:debugger:protocol", {
+    command: "Browser.grantPermissions",
+    params: {
+        permissions: ["clipboardReadWrite", "clipboardSanitizedWrite"],
+        origin: window.location.origin
+    }
+});
+
+context("Export & Import Pages", () => {
+    beforeEach(() => cy.login());
+
+    describe("When exporting and importing page", () => {
+        const pageTitle1 = `Test published page 1`;
+
+        afterEach(() => {
+            cy.pbDeleteSpecificPage(pageTitle1);
+        });
+
+        const searchForPage = title => {
+            cy.findByTestId("default-data-list.search").within(() => {
+                cy.findByPlaceholderText(/Search.../i).type(title);
+            });
+        };
+
+        const clearSearch = () => {
+            cy.findByTestId("default-data-list.search").within(() => {
+                cy.findByPlaceholderText(/Search.../i).clear({ force: true });
+            });
+        };
+
+        it("should be able to export and import a page", () => {
+            cy.visit("/page-builder/pages");
```

Comment:
Since we're not testing the page creation flow/UI here, let's create/publish the page programatically. Gonna be faster.

Example here :
```cypress/integration/admin/pageBuilder/cacheControlHeaders.spec.js:23```

Prolly we can delete all lines until `searchForPage(pageTitle1);` one.